src/bin/bat/assets.rs

```diff
@@ -41,8 +41,14 @@ pub fn assets_from_cache_or_binary() -> Result<HighlightingAssets> {
         }
     }
 
-    Ok(HighlightingAssets::from_cache(&cache_dir)
-        .unwrap_or_else(|_| HighlightingAssets::from_binary()))
+    let no_custom_assets = wild::args_os().any(|arg| arg == "--no-custom-assets");
+    let from_cache = if no_custom_assets {
+        Err("Skipping custom assets due to --no-custom-assets arg".into())
```

Comment:
Took me a while to understand this. Is this just a dummy error in order to take the "else" path in `unwrap_or_else` below? Isn't there another, simpler way to write this logic?

Reply:
I wasn't super happy with it either, but though it was good enough. But if you struggled with it on first try it definitely is not clear enough code. I'll see if I can come up with something clearer.